src/containers/NFT/test/NFT.test.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
Why is this needed?

Reply:
Due to the way the test is mocking `useState`.  I was able to update all the other places though.  Good catch there were 12 places.  I used a script that was supposed to rename all of those but I guess it missed a few.